superset-frontend/src/dataMask/reducer.ts

```diff
@@ -139,8 +139,6 @@ const dataMaskReducer = produce(
           action.filterConfig ?? [],
           cleanState,
           draft,
-          // @ts-ignore
-          action.data.dataMask,
```

Comment:
Do we have the same issue on line 134?

Reply:
No, in line 134 `action` object does have `data.dataMask` field.